www/doc/1.7/report.html

```diff
@@ -20,143 +22,176 @@ <h1>Herbie reports</h1>
     </nav>
   </header>
 
-  
   <h1>The Herbie report</h1>
 
   <p>When used <a href="using-web.html">in the browser</a>, Herbie
-  generates HTML reports full of information about the accuracy of
-  your input and its output expressions.</p>
+    generates HTML reports full of information about the accuracy of
+    your input and its output expressions.</p>
 
   <h2 id="summary">Summary numbers</h2>
 
   <p>First, a brief summary of the results. For most uses, the
-    “Average Error” number, which describes how accurate the input and
-    output expressions are, is the most important statistic in this
-    section. The other numbers list time Herbie took to improve the
-    program and the <a href="input.html#precisions">precision</a>
-    assumed for floating-point operations.
+    "Percentage Accurate" and "Speedup" metric, which describes the change in accuracy and speed up from
+    the starting expression to what Herbie thinks might be the best alterative,
+    is the most important statistics in this section. The other numbers list
+    time Herbie took to improve the program and number of other alternatives
+    found.
```

Comment:
Ok, this is confusing and also maybe incorrect. "Percentage Accurate" refers to the most accurate alternative. The "Speedup" refers to the fastest alternative that's more accurate than the original. Maybe start by talking about the time and number of alternatives, because that will explain to readers that there is more than one alternative (a new feature this release!) and help them understand the explanations of Percentage Accurate and Speedup.

Reply:
Now that I look at this, this paragraph is just duplicating text in the list below. So let's shorten this paragraph.